src/libs/ajax/ajax-common.ts

```diff
@@ -62,22 +62,29 @@ export async function makeRequestRetry(request: Function, retryCount: number, ti
   }
 }
 
+const isUnauthorizedResponse = (error): boolean => error instanceof Response && error.status === 401;
```

Comment:
```suggestion
const isUnauthorizedResponse = (error: unknown): boolean => error instanceof Response && error.status === 401;
```

Nit: where possible, avoid implicit `any`.